ext/language_tools.js

```diff
@@ -1511,9 +1537,9 @@ var Autocomplete = function() {
         }
 
         editor.on("changeSelection", this.changeListener);
-        editor.on("blur", this.blurListener);
-        editor.on("mousedown", this.mousedownListener);
-        editor.on("mousewheel", this.mousewheelListener);
+        //editor.on("blur", this.blurListener);
```

Comment:
Why did you comment these lines?

Reply:
These are for not closing the popup and it stays intact on these events.